app_config_b2c.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+b2c_tenant = "fabrikamb2c"
+signupsignin_user_flow = "b2c_1_signupsignin1"
+editprofile_user_flow = "b2c_1_profileediting1"
+authority_template = "https://{tenant}.b2clogin.com/{tenant}.onmicrosoftonline.com/{user_flow}"
```

Comment:
>user_flow} [](start = 84, length = 10)

cool! user_flow! I think we need to start updating our samples as well.

Reply:
@jennyf19 One issue I see with "user_flow" is that while all user flows are policies, not all policies are user flows--IEF custom policies could be referenced here. Referring to these as "*_policy" would be the more portable option.

![image](https://user-images.githubusercontent.com/13394541/68041846-b395b800-fc8e-11e9-83e7-34db5aff08b6.png)

Reply:
@mmacy we talked w/ @valnav and he suggests we use `user_flow`, but again, we can discuss more next week.

Reply:
@jennyf19 @valnav Another issue with using `user_flow` is that it deviates from the language used in our tooling. For example, the `dotnet new mvc` command calls them policies:

![image](https://user-images.githubusercontent.com/13394541/68230140-5c088c80-ffad-11e9-8e8d-b9355d01382e.png)

As does Visual Studio 2019:

![image](https://user-images.githubusercontent.com/13394541/68230765-8d358c80-ffae-11e9-98e8-4fa8042b771d.png)

Reply:
@mmacy Thanks for the additional information. Very helpful.

Reply:
Renaming an already widely referenced concept from our tons of docs, wikis, samples, decks, shipped products, portal(s) ... would definitely be a huge effort in its own, and there WILL be a (long?) transition period since all those changes would not happen in one atomic PR. @jennyf19 can we get confirmation (from Naveen?) that we can proceed using the "User Flow" concept from now on, regardless of when/whether other materials would eventually be updated?

Reply:
To be clear, in the portal for B2C, there are both **User flows (policies)** and **Custom policies** (accessible via _Identity Experience Framework_), both of which reside under the **Policies** heading:

![image](https://user-images.githubusercontent.com/13394541/68252230-cdf6cb00-ffd9-11e9-88c3-7cb7ef723088.png)

@rayluo are the other artifacts you mention already widely referring to these exclusively as "user flows"? I may (probably) do not have all the context, here, as @valnav may have a bigger plan to converge them all under User Flows, so Jenny is probably right in that we should discuss tomorrow. :wink:

Reply:
Yes, @mmacy  and @rayluo ...we need to discuss. My understanding from @valnav is that even though they are separate, they are actually all user flows under a user journey, and we should be using user flow, but let's discuss further. There is enough confusion out there already, we don't need to make more :) and maybe the time to change is down the road.

FYI: @jmprieur